wp-content/civi-extensions/net.ourpowerbase.qrcodecheckin-2.1.2/CRM/Qrcodecheckin/Page/QrcodecheckinLanding.php

```
@@ -0,0 +1,96 @@
+    else {
+      $this->assign('status_class', 'qrcheckin-status-other');
+    }
+  }
+
+  private function getDisplayName() {
+    $sql = "SELECT display_name FROM civicrm_contact c JOIN civicrm_participant p ON c.id = p.contact_id
+      WHERE p.id = %0";
+    $dao = CRM_Core_DAO::executeQuery($sql, array(0 => array($this->participant_id, 'Integer')));
+    $dao->fetch();
+    return $dao->display_name;
+  }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'getDisplayName'. (Unused Code Rules)

(UnusedPrivateMethod)